cmd/copyS2SMigrationBlobEnumerator.go

```diff
@@ -86,6 +90,9 @@ func (e *copyS2SMigrationBlobEnumerator) enumerate(cca *cookedCopyCmdArgs) error
 	// b: https://<blob-service>/containerprefix*/vd/blob
 	if isAccountLevel, containerPrefix := e.srcBlobURLPartExtension.isBlobAccountLevelSearch(); isAccountLevel {
 		glcm.Info(infoCopyFromAccount)
+		if cca.blobType == cca.blobType.PageBlob() {
+			glcm.Info("Copying entire account to page blobs will fail on files without a content length of modulus 512 bytes")
```

Comment:
Can we tweak this slightly to say "Copying entire account to page blobs will fail on any files where the content length is not a multiple of 512 bytes."

(I.e. 
1. add the word "any" before "files".  Otherwise user may feel we are _telling_ them that their files _are_ mis-sized. (In reality, we are displaying this message without any knowledge of the file sizes).
2. tweak wording to use multiple instead of modulus
)